src/routes/safe/components/Apps/api/fetchSafeAppsList.ts

```diff
@@ -17,5 +17,6 @@ export type AppData = {
 }
 
 export const fetchSafeAppsList = async (): Promise<TokenListResult> => {
-  return axios.get(SAFE_APPS_LIST_URL).then(({ data }) => data)
+  const networkId = getNetworkId()
+  return axios.get(`${SAFE_APPS_LIST_URL}?network_id=${networkId}`).then(({ data }) => data)
```

Comment:
This API will be used for more things in future. It would make sense to rename `fetchSafeAppsList.ts` to `config-service.ts`.

Reply:
It's placed in `src/routes/safe/components/Apps/api/fetchSafeAppsList.ts` though, not sure if it's the place for general config-service.ts stuff

Reply:
Changed the name, should i move the file to other location?

Reply:
Why did you rename it? It makes no sense to store a file for generic config service stuff in `src/routes/safe/components/Apps/api/`

Reply:
So far we decided to store files that are unique to some part of our app close to such part of the app. This file contains a function to fetch an app and the only place where we fetch apps is the apps list, this is why it's placed here.

Reply:
Oh ok, i misunderstood. So i should leave it there and keep the previous name

Reply:
We've discussed this in the standup and decided to move the file to `src/logic/config-service`.
It will be used for feature toggles and other things, so not just for Safe Apps.